src/app/main.cpp

```diff
@@ -235,19 +235,25 @@ int main(int argc, char *argv[])
 #if defined(DISABLE_GUI) && !defined(Q_OS_WIN)
         if (params.shouldDaemonize)
         {
-            app.reset(); // Destroy current application
-            if (daemon(1, 0) == 0)
+            app.reset(); // Destroy current instance
```

Comment:
IMO, "instance" is misleading in this context. Could you at least use "Application instance"? (Because this means exactly that we are deleting the object of Application class that the pointer is currently referring to.)

Reply:
All done.